src/clj/metafacture_playground/process.clj

```diff
@@ -8,15 +8,21 @@
    (org.metafacture.runner Flux)))
 
 (defn- content->tempfile-path [content file-extension]
-  (let [temp-file (File/createTempFile "metafix" file-extension)]
-   (with-open [file (jio/writer temp-file)]
-     (binding [*out* file]
-       (print content)))
-    (.deleteOnExit temp-file)
-    (let [file-path (clj-str/replace (.getAbsolutePath temp-file) "\\" "/")]
-      (log/info "Wrote content to temp file:" file-path)
-      (log/trace "Content" content)
-      file-path)))
+  (let [tmp-directory (-> (System/getProperty "user.dir")
+                          (str File/separator "tmp")
+                          (File.))]
+    (when-not (.exists tmp-directory)
+      (log/info "Created directory: " (.getAbsolutePath tmp-directory))
```

Comment:
I think the message should either be logged after it is created, or the wording should be changed to "Creating...", otherwise on failure on `mkdir`, in the log it will seem like the error occurs after a successful `mkdir`.